sources/packages/api/src/route-controllers/institution/institution.controller.ts

```diff
@@ -140,7 +140,7 @@ export class InstitutionController extends BaseController {
         "The user has no institution associated with.",
       );
     }
-    const institutionUserAndCount = await this.institutionService.allUsers(
+    const [institutionUsers, count] = await this.institutionService.allUsers(
```

Comment:
We can use the PaginatedResults interface that we created here.

Reply:
we can. this is not part of the PR